server/apps/microservices/src/processors/thumbnail.processor.ts

```diff
@@ -49,11 +50,19 @@ export class ThumbnailGeneratorProcessor {
 
     if (asset.type == AssetType.IMAGE) {
       try {
-        await sharp(asset.originalPath, { failOnError: false })
+        await sharp(asset.originalPath, { failOnError: true })
```

Comment:
If you change this to true, some of the conversions will flat out fail instead of converting to its best ability

Reply:
But if it doesn't have that set, it flat out doesn't error and we cannot know it failed and cannot fallback to exiftool.

I'm open to ideas on how to handle that as sharp will only throw a generic error but not a custom one.

Reply:
I don't know. The information that I have to work with is that if failOnError is set to false it will convert to its best ability, otherwise, it would throw an error.

So maybe you can keep this behavior as originally implemented and fall back in the catch block

Reply:
> I don't know. The information that I have to work with is that if failOnError is set to false it will convert to its best ability, otherwise, it would throw an error.
> 
> So maybe you can keep this behavior as originally implemented and fall back in the catch block

It needs to throw an error as the .catch block will then let it the error be handled via exiftool. If exiftool errors for some reason, and the code is in the main catch block (and not in an `.catch`), I don't think anything would catch it?

Reply:
It seems I was wrong about the `failOnError` and set it to false again. the `.catch` does get used with .srw and .raf pictures.